content/building-a-preview/work-with-base-previews/building-new-previews.md

```diff
@@ -65,3 +65,57 @@ Select the {{% ui-text %}}Build with no base preview{{% /ui-text %}} option.
 ![Select Build with no base preview](/_images/select-build-with-no-base-preview.png)
 
 {{% /expand%}}
+
+## Build a Preview from a specific Base Preview
+
+If you want to use a Preview as a Base Preview for a specific Preview build, but don't want to set it as a [Repository]
```

Comment:
[Repository] and [Branch Base Preview] are rendering as is here. Should they be links?

<img width="693" alt="Screen Shot 2020-04-09 at 1 00 25 PM" src="https://user-images.githubusercontent.com/1538360/78926084-18391800-7a62-11ea-9931-fdaff86028d6.png">

Reply:
Aww, it was kind of you to attribute this to a rendering issue. I just missed adding the links ;) Told you my brain was melting. Well spotted, thanks, @bchavet !